piet-coregraphics/Cargo.toml

```
@@ -13,7 +13,7 @@ piet = { version = "0.2.0", path = "../piet" }
 
 core-graphics = "0.19.1"
 core-text = "16.0"
-core-foundation = "0.7"
+core-foundation = "0.7.1"
```

```suggestion
core-foundation = "=0.7.1"
```

If they end up yanking it's possible that 0.7.2 would be just a revert and we would break again, so let's pin exactly to this release until we know what's up.